src/edfi-performance-test/edfi_performance_test/api/client/ed_fi_api_client.py

```diff
@@ -118,6 +118,17 @@ def get_item(self, resource_id):
         self.log_response(response)
         return json.loads(response.text)
 
+    def _get_resource_id_from_location(self, headers: dict) -> Optional[str]:
+        header = ""
+        if "Location" in headers:
+            header = "Location"
+        elif "location" in headers:
+            header = "location"
+        else:
+            return None
+
+        return headers[header].split("/")[-1].strip()
```

Comment:
ODS/API and Meadowlark return different cases. Per MDN: "An HTTP header consists of its case-insensitive name". There's probably a better solution available, like lower-casing all headers and looking for "location". This works for now.